corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
-hqDefine('app_manager/js/forms/case_config_ui', function () {
+hqDefine('app_manager/js/forms/case_config_ui', [
+    'hqwebapp/js/privileges',
```

Comment:
Does this load all privileges of the domain to have them available in the UI?
If yes, would it better to just add to the context if the domain has access to DD and add that as initial page data?

Reply:
The `hasPrivilege()` function which gets called [here](https://github.com/dimagi/commcare-hq/blame/4218209b82c61e68b3e329d470c04154eb250515/corehq/apps/hqwebapp/static/hqwebapp/js/privileges.js#L14) looks through `initialPageData`. A list of all the privileges are already available in JS and we're simply seeing if a specific one exists in the list.

Reply:
Cool, thank you.